src/api.js

```diff
@@ -35,6 +35,15 @@ const patchArticle = (article_id, buttonValue) => {
     })
 }
 
+const postComment = (article_id, comment) => {
+    const postBody = {
```

Comment:
Awesome to see the body declared above the request, makes the code easy to read.